events/20200423121631337-RegisterEndpointEvent.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "RegisterEndpointEvent",
+  "issuedAt": "2020-04-23T12:16:31.337857222Z",
+  "payload": {
+    "organization": "urn:oid:2.16.840.1.113883.2.4.6.1:48000001",
+    "URL": "tcp://nuts-development.ons.io",
+    "endpointType": "urn:nuts:endpoint:consent",
+    "identifier": "urn:ietf:rfc:1779:O=Nedap,C=NL,L=Groenlo,CN=nedap_nuts_cordapp_development",
+    "status": "active"
+  }
```

Comment:
This file is malformed; it's missing the end accolade ( } ). By the way, you can test this by starting the **nuts** binary with this repository as registry datadir. I use something like this:

`NUTS_REGISTRY_DATADIR=<path to registry directory> NUTS_IDENTITY=urn:oid:1.3.6.1.4.1.54851.4:08013836 NUTS_AUTH_ACTINGPARTYCN=foo NUTS_AUTH_PUBLICURL=bar ./nuts`

Reply:
the files which are generated probably lack an ending newline character, as I copy + pasted it here :)